Packs/GuardiCore/README.md

```diff
@@ -0,0 +1,18 @@
+Guardicore delivers easy-to-use Zero Trust network segmentation to security practitioners
+across the globe. Our mission is to minimize the effects of high-impact breaches, like
+ransomware, while protecting the critical assets at the heart of your network. We shut 
+down adversarial lateral movement, fast. From bare metal to virtual machines and containers, 
+Guardicore has you covered across your endpoints, data centers and the cloud. 
+Our software-based platform helps you become more secure to enable your organization’s digital transformation. 
+
+
+
+- East West Segmentation / Micro Segmentation capabilities to reduce your attack surface and remove unnecessary streams of communication between servers and applications. 
+- Improve operational efficiency and agility by eliminating downtime during server or application migrations. 
+- Enforce Zero Trust security protocols with ease and simplify your organization's path to compliance. 
+
+Add helpful, relevant links below 
+- Guardicore Website > https://www.guardicore.com/
+- Guardicore Centra Demo Request > https://www.guardicore.com/lp/demo-request/
+- Link to a page/blog/video you have on this integration?
+- Short video on Mastering Segmentation > https://www.youtube.com/watch?v=unUmrOFEKIU

```

Comment:
```suggestion
#### Helpful, relevant links below
- [Guardicore Website](https://www.guardicore.com/)
- [Guardicore Centra Demo Request](https://www.guardicore.com/lp/demo-request/)
- Link to a page/blog/video you have on this integration?
- [Short video on Mastering Segmentation](https://www.youtube.com/watch?v=unUmrOFEKIU)

```

Reply:
Hi Darya, we can remove line 14 and 17. Thank you.